AENS.md

```diff
@@ -297,18 +303,25 @@ A `claim` transaction MUST NOT be in included in the same block as its
  ------------ ----
 | client_ttl | 8  |
  ------------ ----
+| rent       | 32 |
+ ------------ ----
 | pointers   |    |
  ------------ ----
 ```
 
-The `update` transaction MUST be signed by the owner
-of the name entry to be updated.
+The `update` transaction MUST be signed by the owner of the name entry to be
+updated.
 
-The `expire_by` MUST NOT be more than 36000 blocks into
-the future.
+The `expire_by` MUST NOT be more than `floor(60 * 24 * (365/2)/key_block_interval)`,
```

Comment:
The normative serialization document serializations.md (in this repo) calls this relative TTL `name_ttl`. It may help the reader to link in this document the serializations in serializations.md and use here the same field names as in serializations.md when describing them - as here.

Reply:
I'm actually not sure why it is even called `name_ttl` here and then `expires` in the tree. Seems inconsistent.